bears/python/requirements/PySafetyBear.py

```diff
@@ -40,15 +54,36 @@ class PySafetyBear(LocalBear):
         'Python 3 Requirements',
     }
     AUTHORS = {'Bence Nagy'}
-    REQUIREMENTS = {PipRequirement('safety', '0.5.1')}
+    REQUIREMENTS = {PipRequirement('safety', '1.8.2')}
     AUTHORS_EMAILS = {'bence@underyx.me'}
     LICENSE = 'AGPL'
     CAN_DETECT = {'Security'}
 
-    def run(self, filename, file):
+    def setup_dependencies(self):
+        insecure_full_json_url = ('https://raw.githubusercontent.com/'
+                                  'pyupio/safety-db/master/data/'
+                                  'insecure_full.json')
+        insecure_josn_url = ('https://raw.githubusercontent.com/'
+                             'pyupio/safety-db/master/data/insecure.json')
+        self.download_cached_file(insecure_full_json_url,
+                                  'insecure_full.json')
+        my_file = self.download_cached_file(insecure_josn_url,
+                                            'insecure.json')
+        type(self).db_path = os.path.dirname(my_file)
+
+    def run(self, filename, file,
+            db_path: str = None,
```

Comment:
See other bears (inc CheckstyleBear ) for the type to use for paths

Reply:
I don't get what exactly you are saying, when I initially make it to `db_path: str = ''` you check if it is `None` so I make it to None.

Reply:
got your point :+1:  `from coalib.settings.Setting import path`